### PR TITLE
Run update-packages

### PR DESCRIPTION
**Pull request by jmagman**
Run `flutter update-packages --force-upgrade`

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: packages/flutter_tools/pubspec.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ dependencies:
   args: 2.3.0
   browser_launcher: 1.1.0
   dds: 2.2.0
-  dwds: 12.1.0
+  dwds: 13.0.0
```
Comment (jmagman) [inline packages/flutter_tools/pubspec.yaml:14]: `dwds` major bump 
https://pub.dev/packages/dwds/changelog#1300

**Comment by christopherfujino** [comment]
I'm guessing test failures are legit?

**Comment by jmagman** [comment]
> I'm guessing test failures are legit?

Sure seems it.  @elliette or @annagrin any idea what's up here with the web test timeouts from `dwds: 12.1.0` to `13`?

File: packages/flutter_tools/pubspec.yaml (hunk 1 above)
Comment (annagrin) [reply packages/flutter_tools/pubspec.yaml:14]: thanks Jen, this what I was trying to do now:) That might require changes in devfs_web.dart

**Comment by annagrin** [comment]
> Sure seems it. @elliette or @annagrin any idea what's up here with the web test timeouts from dwds: 12.1.0 to 13?

I am going to sync to this branch and try it out. We have API change so some changes might be needed  at the called of `dsdw.start`.

**Comment by jmagman** [comment]
Closing in favor of https://github.com/flutter/flutter/pull/101450

**Pull request closed without merging**